extensions/mqtt/AbstractMQTTProcessor.cpp

```diff
@@ -119,6 +126,38 @@ void AbstractMQTTProcessor::onSchedule(core::ProcessContext *context, core::Proc
     qos_ = valInt;
     logger_->log_debug("AbstractMQTTProcessor: QOS [%ll]", qos_);
   }
+  value = "";
+
+  if (context->getProperty(SecurityProtocol.getName(), value) && !value.empty()) {
+    if (value == MQTT_SECURITY_PROTOCOL_SSL) {
+      sslEnabled_ = true;
```

Comment:
what happens if a cert isn't specified or the CA isn't specified but needs to be?

Reply:
the SSL handshake will fail and based on the error, user need to config the right certs.